RawWaterManagement/HydraulicSimulationScenario/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,144 @@
+{
+  "id": "urn:ngsi-ld:dataModel:id:GZAR:07210621",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1970-05-23T01:18:44Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2010-12-22T14:03:23Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Water remember theory vote fish. Speak hotel think wish. Nature find speech year may."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Day despite article rich agent deal mouth. Opportunity star set. Wish do generation able us party sell. Reveal approach building technology return movement positive."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "So product organization last."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Interview other plant pattern close policy focus. Well hotel goal ball suddenly third goal."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Heart may put room. Dog product hot reflect expect. Write modern spring industry on summer."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:QJVR:94419439",
+      "urn:ngsi-ld:dataModel:items:WDTQ:88171012"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:JDRW:02822125",
+      "urn:ngsi-ld:dataModel:items:HSTA:26679121"
+    ]
+  },
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -52.853197,
+        -113.959231
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Beat capital real decade agreement issue. Person hundred require answer image could.",
+      "addressLocality": "Generation couple expect environmental simply gas social. While expert my bed thus officer special.",
+      "addressRegion": "Wind want single message. Tend with return often parent. Including effect total sign could industry lay.",
+      "addressCountry": "Toward continue soon treatment. Tree lead spend color base marriage nature. Author statement or until.",
+      "postalCode": "Try experience huge. Mr tax wind all force second once. Money vote positive Democrat.",
+      "postOfficeBoxNumber": "Story account tax admit billion card find. Student sing never true start significant around.",
+      "areaServed": "Television human prepare. Customer likely not firm until."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Represent girl among hotel figure. Big new those politics trip."
+  },
+  "type": "HydraulicSimulationScenario",
+  "hasInputOpenChannelSystem": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}